datalad/support/gitrepo.py

```diff
@@ -1385,6 +1385,18 @@ def get_tracking_branch(self, branch=None):
 
         return track_remote, track_branch
 
+    @property
+    def count_objects(self):
+        """return dictionary with count, size(in KiB) information of git objects
+        """
+        try:
+            count_cmd = ['git', 'count-objects', '-v']
+            count_str, err = self._git_custom_command('', count_cmd)
+            count_list = count_str.replace(': ', '\n').split('\n')
+            count = {key: int(value) for (key, value) in zip(count_list[0::2], count_list[1::2]) if key and value}
```

Comment:
This is overcomplicated solution... you can do in a single dict comprehension, just split each line on :